api/envoy/api/v2/route/route.proto

```diff
@@ -114,6 +114,9 @@ message VirtualHost {
   // Indicates that the virtual host has a CORS policy.
   CorsPolicy cors = 8;
 
+  // Indicates that the route has a CSRF policy.
+  CsrfPolicy csrf = 18;
```

Comment:
Do we need this be in VH and Route directly, not as per_route_config? `CorsPolicy` is here just because they predates per route config.

Reply:
+1 on using per route config for this. Optimally in the future we would deprecate and move the CORS stuff also but not a big deal for now.